README.md

```diff
@@ -39,3 +39,253 @@ Default setup for Ditto uses Ngix with basic authentication. A custom authentica
 in the library to support it. See [BasicAuth Provider](src/ditto_client/basic_auth.py).
 
 [See examples/basic.py for the full usage](examples/basic.py)
+
+## Usage - CLI
+
+The Ditto client includes a comprehensive CLI for interacting with Eclipse Ditto services. The CLI provides the following commands:
+
+| Command Group | Description | Credentials Used |
+|---------------|-------------|------------------|
+| `things` | Manage things (digital twins) | Standard (`ditto:ditto`) |
```

Comment:
Remove the credentials used column from here .... people will use their credentials against their servers. There is no default in the code anyways.